### DISCUSSION TITLE
Synology Immich v1.122.1 new install fails to start:  Failed to create upload/encoded-video/.immich:

### DISCUSSION THREAD

**Discussion by philos31** [immich-app/immich#14531] [Q&A] [answered]
(Use `node --trace-deprecation ...` to show where the warning was created)

(node:17) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

(node:7) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

[Nest] 7  - 12/06/2024, 8:49:32 PM     LOG [Microservices:EventRepository] Initialized websocket server

[Nest] 7  - 12/06/2024, 8:49:32 PM     LOG [Microservices:MetadataService] Bootstrapping metadata service

[Nest] 7  - 12/06/2024, 8:49:32 PM     LOG [Microservices:MetadataService] Initializing metadata service

[Nest] 7  - 12/06/2024, 8:49:32 PM     LOG [Microservices:MapRepository] Initializing metadata repository

[Nest] 17  - 12/06/2024, 8:49:32 PM     LOG [Api:EventRepository] Initialized websocket server

[Nest] 17  - 12/06/2024, 8:49:33 PM     LOG [Api:ServerService] Feature Flags: {

  "smartSearch": true,

[Nest] 17  - 12/06/2024, 8:49:33 PM   ERROR [Api:StorageService] Failed to create upload/encoded-video/.immich: Error: EACCES: permission denied, mkdir 'upload/encoded-video'

api worker exited with code 1

  "facialRecognition": true,

  "duplicateDetection": true,

  "map": true,

  "reverseGeocoding": true,

  "importFaces": false,

  "sidecar": true,

  "search": true,

  "trash": true,

  "oauth": false,

  "oauthAutoLaunch": false,

  "passwordLogin": true,

  "configFile": false,

  "email": false

}

[Nest] 17  - 12/06/2024, 8:49:33 PM     LOG [Api:StorageService] Verifying system mount folder checks, current state: {"mountChecks":{}}

[Nest] 17  - 12/06/2024, 8:49:33 PM     LOG [Api:StorageService] Writing initial mount file for the encoded-video folder

Initializing Immich v1.122.1

Detected CPU Cores: 4

Starting api worker

Starting microservices worker

(node:7) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

(node:7) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

(node:17) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

[Nest] 7  - 12/06/2024, 8:50:00 PM     LOG [Microservices:EventRepository] Initialized websocket server

[Nest] 7  - 12/06/2024, 8:50:00 PM     LOG [Microservices:MetadataService] Bootstrapping metadata service

[Nest] 7  - 12/06/2024, 8:50:00 PM     LOG [Microservices:MetadataService] Initializing metadata service

[Nest] 7  - 12/06/2024, 8:50:00 PM     LOG [Microservices:MapRepository] Initializing metadata repository

[Nest] 17  - 12/06/2024, 8:50:00 PM     LOG [Api:EventRepository] Initialized websocket server

[Nest] 17  - 12/06/2024, 8:50:01 PM     LOG [Api:ServerService] Feature Flags: {

  "smartSearch": true,

  "facialRecognition": true,

  "duplicateDetection": true,

  "map": true,

  "reverseGeocoding": true,

  "importFaces": false,

  "sidecar": true,

  "search": true,

  "trash": true,

  "oauth": false,

  "oauthAutoLaunch": false,

  "passwordLogin": true,

  "configFile": false,

  "email": false

}

[Nest] 17  - 12/06/2024, 8:50:01 PM     LOG [Api:StorageService] Verifying system mount folder checks, current state: {"mountChecks":{}}

[Nest] 17  - 12/06/2024, 8:50:01 PM     LOG [Api:StorageService] Writing initial mount file for the encoded-video folder

[Nest] 17  - 12/06/2024, 8:50:01 PM   ERROR [Api:StorageService] Failed to create upload/encoded-video/.immich: Error: EACCES: permission denied, mkdir 'upload/encoded-video'

api worker exited with code 1

Initializing Immich v1.122.1

Detected CPU Cores: 4

Starting api worker

Starting microservices worker

(node:7) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

(node:17) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

(node:7) [DEP0060] DeprecationWarning: The `util._extend` API is deprecated. Please use Object.assign() instead.

(Use `node --trace-deprecation ...` to show where the warning was created)

[Nest] 7  - 12/06/2024, 8:50:28 PM     LOG [Microservices:EventRepository] Initialized websocket server

[Nest] 7  - 12/06/2024, 8:50:28 PM     LOG [Microservices:MetadataService] Bootstrapping metadata service

[Nest] 7  - 12/06/2024, 8:50:28 PM     LOG [Microservices:MetadataService] Initializing metadata service

[Nest] 7  - 12/06/2024, 8:50:28 PM     LOG [Microservices:MapRepository] Initializing metadata repository

[Nest] 17  - 12/06/2024, 8:50:29 PM     LOG [Api:EventRepository] Initialized websocket server

[Nest] 17  - 12/06/2024, 8:50:29 PM     LOG [Api:ServerService] Feature Flags: {

  "smartSearch": true,

  "facialRecognition": true,

  "duplicateDetection": true,

  "map": true,

  "reverseGeocoding": true,

  "importFaces": false,

  "sidecar": true,

  "search": true,

  "trash": true,

  "oauth": false,

  "oauthAutoLaunch": false,

  "passwordLogin": true,

  "configFile": false,

  "email": false

}

[Nest] 17  - 12/06/2024, 8:50:29 PM     LOG [Api:StorageService] Verifying system mount folder checks, current state: {"mountChecks":{}}

[Nest] 17  - 12/06/2024, 8:50:29 PM     LOG [Api:StorageService] Writing initial mount file for the encoded-video folder

[Nest] 17  - 12/06/2024, 8:50:29 PM   ERROR [Api:StorageService] Failed to create upload/encoded-video/.immich: Error: EACCES: permission denied, mkdir 'upload/encoded-video'

**Comment by alextran1502** [comment]
~~- Have you moved around the folder at `UPLOAD_LOCATION`?~~
~~- Is this a new setup?~~
~~- Can you check if your mount is correctly mounted at `UPLOAD_LOCATION`?~~

Clearly I didn't read the title. Please check the permission where you have the `UPLOAD_LOCATION` pointed to

**Comment by philos31** [comment, marked as answer]
Thanks, that was the case. Did not use the right credentials for the upload location.
Reminder to myself, never try to install things with 2 bacardi-coke in the system on a friday night.
